test/integration/BUILD

```diff
@@ -158,6 +158,23 @@ envoy_cc_test(
     ],
 )
 
+envoy_cc_test(
+    name = "protocol_integration_test",
+    srcs = [
+        "protocol_integration_test.cc",
+    ],
+    shard_count = 3,
```

Comment:
From my own edification, how did you pick these shard counts? Maybe a comment here and elsewhere?

Reply:
Oh, the first time I ran them they were annoyingly long (I default to a 60s timeout for integration tests) so I just sharded them until they were comparable.
Commented in the build file for posterity :-)